invisible_cities/detsim/simulate_S1.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import pandas as pd
+
+from typing import Tuple
+from typing import Callable
+
+from scipy.optimize import brentq
+
+from .. core.core_functions import in_range
+from .. core import system_of_units as units
+
+
+def compute_S1_pes_at_pmts(xs      : np.ndarray,
+                           ys      : np.ndarray,
+                           zs      : np.ndarray,
+                           photons : np.ndarray,
+                           LT      : Callable  )->np.ndarray:
+    """Compute the pes generated in each PMT from S1 photons
+    Parameters:
+        :LT: function
+            The Light Table in functional form
+        :photons: np.ndarray
+            The photons generated at each hit
+        :xs, ys, zs: np.ndarray
+            hit position
+    Returns:
+        :pes: np.ndarray
+            photoelectrons at each PMT produced by all hits.
+    """
+    pes = photons[:, np.newaxis] * LT(xs, ys, zs)
+    pes = np.random.poisson(pes)
+    return pes.T
+
+tau1 = 4.5; c1 = 0.1
```

Comment:
Is there a reason why you opted to make this global?

Reply:
I am not sure if that is the best approax. We could define them as local variables inside `S1_minimize` or forget about defining the variables and put them as numbers directly.